packages/blade/src/components/Typography/Code/Code.tsx

```diff
@@ -82,33 +133,38 @@ const CodeContainer = styled(BaseBox)<CodeContainerProps>((props) => {
  * In React Native, you would have to align it using flex to make sure the Code and the surrounding text is correctly aligned
  *
  * ```tsx
- *  <BaseBox flexWrap="wrap" flexDirection="row" alignItems="flex-start">
+ *  <Box flexWrap="wrap" flexDirection="row" alignItems="flex-start">
  *   <Text>Lorem ipsum </Text>
  *   <Code>SENTRY_TOKEN</Code>
  *   <Text> normal text</Text>
- * </BaseBox>
+ * </Box>
  * ```
  */
-
 const Code = ({
   children,
   size = 'small',
   weight = 'regular',
-  color = 'surface.text.subtle.lowContrast',
+  isHighlighted = true,
+  color,
   testID,
   ...styledProps
 }: CodeProps): JSX.Element => {
   const { fontSize, lineHeight } = getCodeFontSizeAndLineHeight(size);
+  const codeTextColor = React.useMemo<CodeProps['color']>(
```

Comment:
why do we use `useMemo` here? any perf things we witnessed? 🤔

Reply:
@saurabhdaware sorry asking on a merged PR but checking all the things that got merged last week 🙈